src/components/TimeCardPage/CellTypes/ConflictableTimeEntry.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from "react";
+import { RowSchema, TimeRowEntry } from "../../../schemas/RowSchema";
+import { TimeEntry } from "./TimeEntry";
+import {
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  Editable,
+  EditablePreview,
+  EditableInput,
+  Input,
+  Box,
+  ButtonGroup,
+  Flex,
+  useDisclosure,
+  useEditableControls,
+  StackDivider,
+  HStack,
+  VStack,
+  Text,
+  IconButton
+} from "@chakra-ui/react";
+
+import {
+  ChatIcon,
+  CheckIcon,
+  CloseIcon,
+  EditIcon,
+  AddIcon,
+  DeleteIcon,
+  WarningIcon
+} from "@chakra-ui/icons";
+
+
+
+interface ConflicatableTimeEntryProps {
+  field: string;
+  row: RowSchema;
+  updateFields: Function;
+}
+
+/**
+ * This represents a time entry cell that accounts for multiple time entries (i.e. an associate's and a supervisor's entries)
+ * and allows for conflicts to be shown.
+ */
+export function ConflicatableTimeEntry(props: ConflicatableTimeEntryProps) {
+  const [selectedTime, setSelectedTime] = useState(null);
+
+  const handleTimeSelection = (time: number) => {
+    setSelectedTime(time);
+  };
+
+  // determine if the associate and supervisor times are conflicting for this cell
+  const hasAssociateTime: boolean = props.row.Associate !== undefined;
+  const hasSupervisorTime: boolean = props.row.Supervisor !== undefined;
+  const hasAdminTime: boolean = props.row.Admin !== undefined;
+
+  // TODO :
+  //  -- only show conflict button on conflict
+  //  -- auto-populate with the following rules:
+  //  ---- if associate and supervisor times are the same, auto-populate with that time
+  //  ---- if associate and supervisor times are different, auto-populate with null
+  //  -- make sure that time conflict entry cells only show up on admins 
+
+  // Conflicts are defined as:
+  // 1. associate submitted a time, and supervisor did not
+  // 2. Supervisor submitted a time for this entry, and associate did not
+  // 3. Associate time and supervisor reported time differ
+  var hasConflict = hasAssociateTime !== hasSupervisorTime || 
```

Comment:
possible issue of associate has time and supervisor doesnt which shouldn't lead to a conflict